src/model_editor/InspectorGadget.cpp

```diff
@@ -794,7 +794,7 @@ void InspectorGadget::layoutComboBox(QVBoxLayout* layout, QWidget* parent, opens
     idx = 0;
     combo->insertItem(idx, QPixmap(":/images/alert_image.png"), curVal.c_str(), "Invalid");
 
```

Comment:
Not sure what happened here, theres no change

Reply:
There's a white space change on line 797, I bet your editor is set to automatically update the indents.  You can just `git checkout HEAD -- src/model_editor/InspectorGadget.cpp`

Reply:
I applied that fix and oddly this still shows up here, that is weird.